app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Admin
+  class UsersController < AdminController
+  before_action :set_user, only: %i[show edit update confirm_owner destroy]
+
+    def index
+      @users = User.all
+    end
+
+    def show; end
+
+    def edit
+      render 'shared/users/edit'
+    end
+
+    def update
+      @current_user = current_user
```

Comment:
Why do you need @current_user variable here? current_user will be available in view.